Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,296 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct CreateBuffersAroundPointsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: model.map, graphicsOverlays: model.graphicsOverlays)
+            .onSingleTapGesture { _, mapPoint in
+                // Update tapPoint and bring up input box if point is within bounds.
+                if model.isWithinBounds(mapPoint) {
+                    model.tapPoint = mapPoint
+                    model.inputBoxIsPresented.toggle()
+                } else {
+                    model.status = .outOfBoundsTap
+                }
+            }
+            .overlay(alignment: .top) {
+                Text(getStatusText(model.status))
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    // Union toggle switch.
+                    Toggle("Union", isOn: $model.shouldUnion)
+                        .toggleStyle(.switch)
+                        .onChange(of: model.shouldUnion) { _ in
+                            if !model.bufferPoints.isEmpty {
+                                model.drawBuffers()
+                            }
+                        }
+                    // Clear button.
+                    Button("Clear") {
+                        model.clearBufferPoints()
+                        model.status = .addPoints
+                    }
+                    .disabled(model.bufferPoints.isEmpty)
+                }
+            }
+            .alert("Buffer Radius", isPresented: $model.inputBoxIsPresented, actions: {
+                TextField("100", text: $model.radiusInput)
+                    .keyboardType(.numberPad)
+                // Input alert done button.
+                Button("Done") {
+                    model.addBufferPoint()
+                    model.radiusInput = ""
+                    model.drawBuffers()
+                    model.status = Status.bufferCreated
+                }
```

Comment:
Please add input validation here (`radiusInput` > 0 && `radiusInput` < 300). Pressing the "Done" button with a radius less than or equal to zero will add a point to the map with no buffer which is unexpected, and will also add a buffer with a radius of any size when the radius is greater than 300 which is also unexpected behavior.